Ska/Sun/tests/test_sun.py

```diff
@@ -3,7 +3,15 @@
 import numpy as np
 from Quaternion import Quat
 from ..sun import (apply_sun_pitch_yaw, get_sun_pitch_yaw, nominal_roll,
-                   off_nominal_roll, position, pitch as sun_pitch)
+                   off_nominal_roll, position, allowed_rolldev,
+                   pitch as sun_pitch)
+
+
+def test_allowed_rolldev():
+    dev1 = allowed_rolldev(135)
+    assert np.isclose(dev1, 13.979)
+    dev2 = allowed_rolldev(138)
+    assert np.isclose(dev2, 14.516)
 
```

Comment:
Add a test for 0, 40, 180.0 and 181.0 deg.

Reply:
And what values are the right answers for 180 and 181 (both greater than the last entry in the table).

Reply:
The right answers are 0.0.

Reply:
I thought the right answer for 180 was still technically the same as the last row in the file (close to but not equal to 180), but suppose that's in the noise.

Reply:
Whether it is 0. for pitch >= 180.0 or pitch > 180 doesn't matter, but certainly for pitch > 180 it should be 0. Whatever ends up being the cleanest implementation can drive the answer there. I would think about adding a value of 0 to the table at 180 so the code is cleaner, but whatever works out.

Reply:
I cheated and just added the value at 180 in the code for now.  I'm not sure if that may be cleaner overall than adding a value to the data file.